sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -177,9 +179,10 @@ public string ReplacementHost
         {
         };
 
-        protected RecordedTestBase(bool isAsync, RecordedTestMode? mode = null) : base(isAsync)
+        protected RecordedTestBase(bool isAsync, RecordedTestMode? mode = null, string apiVersion = null) : base(isAsync)
```

Comment:
We should also document this constructor and parameters explaining the usage and that it is only currently used for mgmt plane.

Reply:
Should we also consider creating a separate ctor overload that takes this argument, just to make it less visible to data plane test classes?

Reply:
This is not intended to be a mgmt plane feature, it allows you record / playback specific versions where as before the only functionality was to be able to run live tests on multiple versions.  Which SDKs opt in or out today shouldn't sway our design since that will change over time.

Reply:
This was resolved when we were fixing a consistency issue between two settings so resolving as no-op at this point.